### PR TITLE
Add iml-timer service to rpm builds

### PR DESCRIPTION
**Pull request by johnsonw**
Add iml-timer service to rpm builds

Signed-off-by: johnsonw <wjohnson@whamcloud.com>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/2250)

### CODE REVIEW

File: iml-job-scheduler.service [hunk 1]
```diff
@@ -3,12 +3,15 @@ Description=IML Job Scheduler Service
 PartOf=iml-manager.target
 After=rabbitmq-server.service
 After=postgresql-9.6.service
+After=iml-settings-populator.service
+Requires=iml-settings-populator.service
```
Comment (jgrund) [inline iml-job-scheduler.service:7]: This shouldn't be needed.

File: iml-job-scheduler.service [hunk 2]
```diff
@@ -3,12 +3,15 @@ Description=IML Job Scheduler Service
...
 
 [Service]
 Type=simple
+EnvironmentFile=/var/lib/chroma/iml-settings.conf
```
Comment (jgrund) [inline iml-job-scheduler.service:11]: This shouldn't be needed.

File: iml-timer.service [hunk 3]
```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=IML Timer Service
+PartOf=iml-manager.target
+After=rabbitmq-server.service
+After=iml-settings-populator.service
+Requires=iml-settings-populator.service
+
+[Service]
+Type=simple
+Environment=RUST_LOG=info,sqlx::query=warn
```
Comment (jgrund) [inline iml-timer.service:10]:
```suggestion
Environment=RUST_LOG=info
```

`sqlx` is not being used here

File: rust-iml.spec [hunk 4]
```diff
@@ -393,7 +395,7 @@ Group: System Environment/Libraries
 Summary: Consumer of snapshot listing
 License: MIT
 Group: System Environment/Libraries
-Requires: rust-iml-agent-comms
+Requires: rust-iml-agent-comms, rust-iml-timer
```
Comment (jgrund) [inline rust-iml.spec:398]:
```suggestion
Requires: rust-iml-agent-comms
```

This package does not have a direct requirement on the timer.

**Review by mkpankov — DISMISSED** [review]
> Reviewed 14 of 17 files at r1, 3 of 3 files at r2.
> *[Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/2250)* status: all files reviewed, 4 unresolved discussions (waiting on @johnsonw)

File: iml-timer.service [hunk 5]
```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=IML Timer Service
+PartOf=iml-manager.target
+After=rabbitmq-server.service
```
Comment (jgrund) [inline iml-timer.service:4]: nit: this does not depend on rabbit

Comment (johnsonw) [reply iml-timer.service:4]: Yeah I guess it's not needed. I thought we would want to wait until afterwards. I'll remove.

**Review by jgrund — APPROVED** [review]

File: chroma-manager.conf.template [hunk 6]
```diff
@@ -405,8 +405,6 @@ server {
         proxy_set_header Http-Host $http_host;
         proxy_pass {{UPDATE_HANDLER_PROXY_PASS}};
     }
-
-    {{INCLUDES}}
```
Comment (ip1981) [inline chroma-manager.conf.template:409]: 👍

**Review by ip1981 — APPROVED** [review]

**Pull request merged**